docs/4.0/trustedclusters.md

```diff
@@ -219,6 +236,17 @@ role name and use reference it to name the local role:
 **NOTE:** The regexp matching is activated only when the expression starts
 with `^` and ends with `$`
 
+
+### Trusted Cluster UI
+For customers using Teleport Pro, they can easily configure _leaf_ nodes using the 
+Teleport Proxy UI. 
+
+**Creating Trust from the Leaf node to the root node.**
+![Tunnels](img/trusted-clusters/setting-up-trust.png)
+
+**The root Cluster showing the Cluster switching UI.** 
```

Comment:
Inconsistency with other use of `Cluster` vs `cluster` in the docs here. I'd probably make it all lower case `cluster` for ease.